### PR TITLE
Return a blocking error if no camera and upload fallback is disabled

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
At the moment, if camera is not present or the browser does not support camera apis, and the upload fallback has been disabled, the user sees a generic error that tells them to go back to desktop even if the flow started on mobile, as non cross-device.
 
# Solution
Display a more informative camera failure message if device camera is not working, not accessible or no present AND upload is disabled. The error message will only be displayed on mobile, because on desktop, when camera cannot be used, the user will be redirected to cross device flow. Must work on cross device and when the flow is started on mobile.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [x] Have new manual tests been written down?
- [x] Have tests passed locally?
- [x] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-566-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Capture/Face.js [hunk 1]
```diff
@@ -77,29 +77,29 @@ class Face extends Component {
...
         <Uploader
           {...props}
           onUpload={ this.handleUpload }
           title={ translate('capture.face.upload_title') || title }
           instructions={ translate('capture.face.instructions') }
-          />
-      :
-        <GenericError error={{name: 'GENERIC_CLIENT_ERROR'}}/>
+          /> :
+          (!isDesktop || this.props.mobileFlow) && <GenericError error={{name: 'INTERRUPTED_FLOW_ERROR'}} />
```
Comment (stephencookdev) [inline src/components/Capture/Face.js]: So desktop users just get... an empty screen?

Comment (stefaniacardenas) [reply src/components/Capture/Face.js]: Desktop users are redirected to x-device if camera cannot be used

Comment (stephencookdev) [reply src/components/Capture/Face.js]: Ah, yes

In that case - why do these checks at all? Why not just omit the `(!isDesktop || this.props.mobileFlow)` bit and always render the error at this bit?

Maybe with a comment about the x-device flow for desktop

Comment (stefaniacardenas) [reply src/components/Capture/Face.js]: Just me over-engineering things!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://566-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/GenericError/assets/interrupted-flow.svg [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (josokinas) [inline src/components/GenericError/assets/interrupted-flow.svg]: I hope you handle SVG compression with webpack, or you can use e.g. https://imageoptim.com/mac

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-566-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://566-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**